fair/ancil/units.py

```diff
@@ -0,0 +1,10 @@
+import numpy as np
+import os
+
+units_filename = os.path.join(os.path.dirname(__file__), 'units.csv')
+
+class Units(dict):
```

Comment:
```suggestion
class Units(dict):
    # TODO: update this so it only loads the csv once rather than once on every instantiation
```

Let's address this later